### PR TITLE
[CN-1202] Revisit Connect to Hazelcast from Outside Kubernetes Tutorial

### PR DESCRIPTION
**Pull request by semihbkgr**

### CODE REVIEW

File: docs/modules/ROOT/pages/hazelcast-platform-operator-expose-externally.adoc [hunk 1]
```diff
@@ -16,7 +16,9 @@
 
 * Up and running https://kubernetes.io/[Kubernetes] cluster
 * Kubernetes command-line tool, https://kubernetes.io/docs/tasks/tools/#kubectl[kubectl]
-* Deployed xref:operator:ROOT:index.adoc[Hazelcast Platform Operator]
+* Deployed xref:operator:ROOT:get-started.adoc[Hazelcast Platform Operator
```
Comment (cagric0) [inline docs/modules/ROOT/pages/hazelcast-platform-operator-expose-externally.adoc:19]:
```suggestion
* Deployed xref:operator:ROOT:get-started.adoc[Hazelcast Platform Operator]
```

File: docs/modules/ROOT/pages/hazelcast-platform-operator-expose-externally.adoc [hunk 2]
```diff
@@ -83,13 +85,15 @@ The `ADDRESS` column displays the external address of your Hazelcast cluster.
...
-To access all examples (excluding CLC), clone the following repository.
+To access all sample clients, clone the following repository:
+
 [source,shell]
 ----
 git clone https://github.com/hazelcast-guides/hazelcast-platform-operator-expose-externally.git
 cd hazelcast-platform-operator-expose-externally
 ----
 
+. The sample code(excluding CLC) for this tutorial is in the link:{github-directory}/tree/master/docs/modules/ROOT/examples[`docs/modules/ROOT/examples/`] directory.
```
Comment (cagric0) [inline docs/modules/ROOT/pages/hazelcast-platform-operator-expose-externally.adoc:96]:
```suggestion
. The sample code(excluding CLC) for this tutorial is in the link:https://github.com/hazelcast-guides/hazelcast-platform-operator-expose-externally/tree/master/docs/modules/ROOT/examples[`docs/modules/ROOT/examples/`] directory.
```
I have checked existing tutorials and this `{github-directory}` now working so I put static link in other tutorials.

**Review by cagric0 — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/hazelcast-platform-operator-expose-externally.adoc (hunk 2 above)
Comment (semihbkgr) [reply docs/modules/ROOT/pages/hazelcast-platform-operator-expose-externally.adoc:96]: did you mean `{github-directory}` isn't working anymore?

File: docs/modules/ROOT/pages/hazelcast-platform-operator-expose-externally.adoc [hunk 3]
```diff
@@ -42,14 +44,7 @@ Run the following command to create the Hazelcast cluster with Expose Externally
 [source, shell]
 ----
 cat <<EOF | kubectl apply -f -
```
Comment (cagric0) [inline docs/modules/ROOT/pages/hazelcast-platform-operator-expose-externally.adoc:46]:
```suggestion
kubectl apply -f - <<EOF
```

File: docs/modules/ROOT/pages/hazelcast-platform-operator-expose-externally.adoc [hunk 4]
```diff
@@ -359,15 +311,7 @@ Run the following command to create the Hazelcast cluster with Expose Externally
...
```
Comment (cagric0) [inline docs/modules/ROOT/pages/hazelcast-platform-operator-expose-externally.adoc:313]:
```suggestion
kubectl apply -f - <<EOF
```

**Review by cagric0 — CHANGES_REQUESTED** [review]

**Pull request merged**